CHANGELOG.md

```diff
@@ -1,5 +1,370 @@
 # Embroider Changelog
 
+## Release (2025-02-11)
+
+@embroider/addon-dev 8.0.0-alpha.0 (major)
+@embroider/addon-shim 2.0.0-alpha.0 (major)
+@embroider/babel-loader-9 4.0.0-alpha.0 (major)
+@embroider/broccoli-side-watch 1.1.0-alpha.0 (minor)
+@embroider/compat 4.0.0-alpha.0 (major)
+@embroider/config-meta-loader 1.0.0-alpha.0 (major)
+@embroider/core 4.0.0-alpha.0 (major)
+@embroider/hbs-loader 4.0.0-alpha.0 (major)
+@embroider/macros 1.17.0-alpha.0 (major)
```

Comment:
```suggestion
@embroider/macros 1.17.0-alpha.0 (minor)
```

Reply:
we talked about this before and we said that we would not change the impact when we're altering the version manually. It's true that release-plan thinks that the this should be a major and that is reflected in the changelog 👍